nautobot/core/templatetags/buttons.py

```diff
@@ -124,6 +126,141 @@ def job_import_url(content_type):
     return import_url
 
 
+def render_tag_attrs(attrs_dict):
+    """Converts tag attributes from a dictionary to a string format suitable for HTML rendering."""
+    return format_html_join(" ", '{}="{}"', [(key, value) for key, value in attrs_dict.items()])
+
+
+@register.inclusion_tag("buttons/consolidated_bulk_action_buttons.html")
+def consolidate_bulk_action_buttons(request, content_type, perms, bulk_edit_url, bulk_delete_url, permissions):
```

Comment:
Can/should we update the signature to only require the request and content type? We can derive permissions from the request.user and the bulk urls from the content type?

Reply:
I think we should make that change. It would reduce the amount of inputs on the templates. As far as having request be in input, I think this should be changed to using `context`, and change the `inclusion_tag` to include `takes_context=True`.

```
@register.inclusion_tag("buttons/consolidated_bulk_action_buttons.html", takes_context=True)
def consolidate_bulk_action_buttons(context, content_type):
```